libse/SubtitleFormats/Utx.cs

```diff
@@ -53,37 +56,37 @@ public override void LoadSubtitle(Subtitle subtitle, List<string> lines, string
         {
             _errorCount = 0;
             subtitle.Paragraphs.Clear();
-            var text = new StringBuilder();
+            var sb = new StringBuilder();
+            char[] splitChars = { '#', ',' };
+            char[] splitChars2 = { ':', '.' };
             for (int i = 0; i < lines.Count; i++)
             {
                 string line = lines[i].Trim();
-
-                if (line.StartsWith('#'))
+                if (line.StartsWith('#') && RegexTimeCode.IsMatch(line))
                 {
-                    var timeParts = line.Split(new[] { '#', ',' }, StringSplitOptions.RemoveEmptyEntries);
-                    if (timeParts.Length == 2)
+                    var timeParts = line.Split(splitChars, StringSplitOptions.RemoveEmptyEntries);
+                    try
                     {
-                        try
-                        {
-                            TimeCode start = DecodeTimeCode(timeParts[0]);
-                            TimeCode end = DecodeTimeCode(timeParts[1]);
-                            subtitle.Paragraphs.Add(new Paragraph(start, end, text.ToString().Trim()));
-                        }
-                        catch
-                        {
-                            _errorCount++;
-                        }
+                        string[] startParts = timeParts[0].Split(splitChars2, StringSplitOptions.RemoveEmptyEntries);
+                        string[] endParts = timeParts[1].Split(splitChars2, StringSplitOptions.RemoveEmptyEntries);
+                        TimeCode start = DecodeTimeCodeFramesFourParts(startParts);
+                        TimeCode end = DecodeTimeCodeFramesFourParts(endParts);
+                        subtitle.Paragraphs.Add(new Paragraph(start, end, sb.ToString().Trim()));
+                    }
+                    catch
+                    {
+                        _errorCount++;
                     }
                 }
                 else if (line.Length > 0)
                 {
-                    text.AppendLine(line.Trim());

```

Comment:
line.Trim() is redundant here, line is already trimmed https://github.com/SubtitleEdit/subtitleedit/pull/1587/files#diff-a1759a18bf40f7264007c4b7775a2160R64